src/thunderbots/software/ai/passing/pass_generator.h

```diff
@@ -108,8 +124,12 @@ namespace AI::Passing
 
         // Weights used to normalize the parameters that we pass to GradientDescent
         // (see the GradientDescent documentation for details)
+        // These weights are *very* roughly the step that gradient descent will take
+        // in each respective dimension for a single iteration. They are tuned to
+        // ensure passes converge as fast as possible, but are also as stable as
+        // possible
         static constexpr double PASS_SPACE_WEIGHT                          = 0.01;
-        static constexpr double PASS_TIME_WEIGHT                           = 0.1;
+        static constexpr double PASS_TIME_WEIGHT                           = 0.01;
```

Comment:
I thought the `PASS_SPEED_WEIGHT` was going to be changed, not the `PASS_TIME_WEIGHT` to improve convergence?

Reply:
Oh whoops, yup, totally correct. Reverted change to `PASS_TIME_WEIGHT` and changed reduced `PASS_SPEED_WEIGHT` by a factor of 10.